wger/manager/templates/set/add.html

```diff
@@ -62,6 +65,48 @@
 automatically becomes "4 x 10".{% endblocktrans %}</span>
         </div>
     </div>
+    <div onclick="revealSettings()">
+        Drop Set: <input type="checkbox" id="set-check">
+    </div>
+
+    <div class="form-group set-items" style="display: none;">
+        <div style="margin-left: 25%;">
+        <span class="help-block">
+        {% blocktrans %}
+            Drop sets are a great way to increase muscle mass.<br/>
+            <strong>Percentage</strong> - Drop the weight by a percentage.<br/>
+            <strong>Cycles</strong> - Number of cycles on the same weight.<br/>
+            <strong>Weight</strong> -  Size of weight to start with.<br/>
+        {% endblocktrans %}
+        </span>
+        </div>
+        <div style="max-width: 45%; margin: auto;">
+          <div class="form-row">
+            <div class="form-group">
+                <label for="exampleFormControlSelect1">Percentage Drop</label>
```

Comment:
The `for=""` in the `<label>` here should match the `id=""` of the `<select>` it is *labelling*.

**Example:**
You have:
```
...
<label for="exampleFormControlSelect1">Percentage Drop</label>
    <select class="form-control" id="percentage-change" ...>
...
```

It should be:
```
...
<label for="percentage-change">Percentage Drop</label>
    <select class="form-control" id="percentage-change" ...>
...
```

Do the same for the rest of the elements in the form.

Reply:
For more reference on the same, see [this](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#Attributes) and [this](https://stackoverflow.com/questions/18432376/what-does-for-attribute-do-in-html-label-tag?answertab=votes#tab-top)